### DISCUSSION TITLE
AdGuard Home - Youtube Ads

### DISCUSSION THREAD

**Discussion by jablochniyvorishka** [AdguardTeam/AdGuardHome#3501] [General]
Hi all.

Im installed Adguard Home in AWS server, and configured my MikroTik to use only one DNS, that is on my AWS server.

On my PC, not all ads is blocking while im watching youtube. On my TV and iPhone via Youtube app also have ads. 

How i can fix it with AdGuard Home?

**Comment by ameshkov** [comment]
Unfortunately, YT ads cannot be blocked on the DNS level.
